src/olympia/editors/templates/editors/home.html

```diff
@@ -58,7 +58,11 @@
             <table>
               {% for row in reviews_total: %}
               <tr>
+                {% if row['user__display_name'] != '' %}
                 <td>{{ row['user__display_name'] }}</td>
+                {% else %}
+                <td>{{ row['user__username'] }}</td>
+                {% endif %}
```

Comment:
Could you try `{{ row['user__display_name'] or row['user__username'] }}` instead?